numba_dppy/dppy_passes.py

```diff
@@ -3,6 +3,7 @@
 import warnings
```

Comment:
Does it make sense to organize all of our passes into a submodule?

Reply:
It does, created issue to track that. https://github.com/IntelPython/numba-dppy/issues/58